examples/todomvc/containers/VisibleTodoList.js

```diff
@@ -0,0 +1,40 @@
+import { connect } from 'react-redux'
+import {  
+  completeTodo,
+  deleteTodo, 
+  editTodo,
+  completeAll
+} from '../actions'
+import TodoList from '../components/TodoList'
+import { getCompletedCount, getVisibleTodos } from '../selectors'
+
+const mapStateToProps = (state) => {
+  return {
+    todos: getVisibleTodos(state),
+    completedCount: getCompletedCount(state)
+  }
+}
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    onDeleteTodo: (id) => {
+      dispatch(deleteTodo(id))
+    },
+    onEditTodo: (id, text) => {
+      dispatch(editTodo(id, text))
+    },
+    onCompleteTodo: (id) => {
+      dispatch(completeTodo(id))
+    },
+    onCompleteAll: () => {
+      dispatch(completeAll())
+    }
+  }
+}
+
+const VisibleTodoList = connect(
+  mapStateToProps,
+  mapDispatchToProps
```

Comment:
You can pass an object as second argument instead of `mapDispatchToProps`. It’s a documented shortcut and has the same effect :wink: